README.md

```diff
@@ -1,9 +1,11 @@
 # MyDataHelpsUI
 
-MyDataHelpsUI is a set of reusable React-based components based on the MyDataHelps.js SDK: https://developer.mydatahelps.org/sdk/
+MyDataHelpsUI is a set of reusable React-based components based on the [MyDataHelps.js SDK](https://developer.mydatahelps.org/sdk/). Explore the component library at [ui.mydatahelps.org](https://ui.mydatahelps.org/).
```

Comment:
We should definitely link to the repository itself (so they can browse the source code) from ui.mydatahelps.org storyboard site

Reply:
(and have a lot more content on the landing page for ui.mydatahelps.org ...right now they are dropped in there but there is no real indication for how to use it etc.)

Reply:
Oops...i'm an idiot...I did not realize what repo I am in.  Sorry...ignore these comments^